ui/app/components/ui/dropdown/dropdown.js

```diff
@@ -0,0 +1,63 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+const Dropdown = ({ className, disabled, name, onChange, options, selectedOption, style }) => {
+  const _onChange = useCallback(
+    (event) => {
+      event.preventDefault()
+      event.stopPropagation()
+      onChange(event.target.value)
+    },
+    [onChange],
+  )
+
+  return (
+    <select
+      className={classnames('dropdown', className)}
+      disabled={disabled}
+      name={name}
```

Comment:
This attribute also doesn't make sense outside of a form context apparently. I was thinking this might be useful for accessibility reasons, but apparently not. Guidelines seem to suggest using a `<label>` element to describe the purpose of each `<select>` element, and then to [use the `title` attribute as a fallback](https://www.w3.org/TR/WCAG-TECHS/H65.html).

I'll update this component to replace `name` with `title`.

Reply:
This has been done in c7b96d47f6b2375d57a103f17f809d00d470931f